packages/runtime/client/rest/src/index.ts

```diff
@@ -105,7 +105,13 @@ export interface RestAPI {
     contractId: ContractId,
     range?: TransactionsRange
   ): Promise<Transactions.GetTransactionsForContractResponse>;
-  //   createTransactionForContract: Transactions.POST; // - https://docs.marlowe.iohk.io/api/create-transactions // TODO: Jamie, lets unify names
+  /**
+   * Create an unsigned transaction which applies inputs to a contract.
+   * @see {@link https://docs.marlowe.iohk.io/api/apply-inputs-to-contract}
+   */
+  applyInputsToContract(
```

Comment:
`applyInputsBuildUnsignedTx` what do you think @hrajchert  ?

I have this in the ts-sdk : 
````
  /**
   * Submit to the Cardano Ledger, the Transaction(Tx) that will apply inputs to a given created contract.
   * @param contractId Contract Id where inputs will be applied
   * @param applyInputsRequest inputs to apply
   * @throws DecodingError
   */
  applyInputs(
    contractId: ContractId,
    applyInputsRequest: ApplyInputsRequest
  ): Promise<TxId>;
```

Reply:
Currently it's mirroring the name used in the Runtime Rest API documentation ([applyInputsToContract](https://docs.marlowe.iohk.io/api/apply-inputs-to-contract)).